src/simulator_core/entities/assets/production_cluster.py

```diff
@@ -111,15 +110,15 @@ def __init__(
         # DemandCluster pressure specifications
         self.pressure_supply = pressure_supply
         self.control_mass_flow = control_mass_flow
-        self._controlled_mass_flow = heat_demand_and_temperature_to_mass_flow(
-            thermal_demand=self.thermal_production_required,
-            temperature_supply=self.temperature_supply,
-            temperature_return=self.temperature_return,
-            pandapipes_net=self.pandapipes_net,
-        )
+#        self._controlled_mass_flow = heat_demand_and_temperature_to_mass_flow(
+#            thermal_demand=self.thermal_production_required,
+#            temperature_supply=self.temperature_supply,
+#            temperature_return=self.temperature_return,
+#            pandapipes_net=self.pandapipes_net,
+#        )
         # Objects of the asset
         self._initialized = False
-        self._create()
+        #self._create()
```

Comment:
Add comment

Reply:
Create function is no longer called from constructor